scripts/restore

```diff
@@ -53,14 +55,15 @@ ynh_restore_file --origin_path="/etc/nginx/conf.d/$domain.d/$app.conf"
 #=================================================
 # RESTORE THE APP MAIN DIR
 #=================================================
-ynh_script_progression --message="Restoring the app main directory..." --time --weight=1
+ynh_script_progression --message="Restoring the app main directory..." --weight=2
 
 ynh_restore_file --origin_path="$final_path"
+ynh_restore_file --origin_path="/var/lib/jellyfin"
```

Comment:
```suggestion
ynh_restore_file --origin_path="/var/lib/$app"
```

Reply:
Same here, I would prefer to leave it explicitly to `/var/lib/jellyfin` since we have no choice.